lib/assessment.sql

```diff
@@ -347,6 +389,8 @@ WITH
         JOIN assessment_questions AS aq ON (aq.id = iq.assessment_question_id)
         JOIN questions AS q ON (q.id = aq.question_id)
         LEFT JOIN users AS u ON (u.user_id = gj.auth_user_id)
+        LEFT JOIN rubric_gradings AS mrg ON (mrg.id = gj.manual_rubric_grading_id)
+        LEFT JOIN rubric_gradings AS arg ON (mrg.id = gj.auto_rubric_grading_id)
```

Comment:
```suggestion
        LEFT JOIN rubric_gradings AS mrg ON (mrg.id = gj.manual_rubric_grading_id)
        LEFT JOIN rubric_gradings AS arg ON (arg.id = gj.auto_rubric_grading_id)
```
possible copy-paste error here?

Reply:
Great catch, thanks. Fixed in cc5d61e.